client/my-sites/plugins/plans/index.tsx

```diff
@@ -51,6 +65,26 @@ const Plans = () => {
 				subHeaderText={ `Choose the plan that's right for you and reimagine what's possible with plugins` }
 				brandFont
 			/>
+			<div className="plans">
+				<PlansFeaturesMain
+					site={ selectedSite || {} } // `PlanFeaturesMain` expects a default prop of `{}` if no site is provided
```

Comment:
We should be able to assume a site is selected here - we redirect [if we have no selected site](https://github.com/Automattic/wp-calypso/blob/5ee25a2101edd07682c9be86ecec84acb322385e/client/my-sites/plugins/index.web.js#L96). (I think that's the middleware that does it)